docs/overview/07_liquidity-amo.md

```diff
@@ -0,0 +1,74 @@
+
+*Note this vault is custom implementation which doesn't rely on the ERC20 nor the ERC1155 standards. Therefore, despite the claim on the LP is tracked in the contracts, users won't receive a standard receipt of the LP tokens in exchange.
+
+### Withdraw
+Users can also withdraw their assets from the vault whenever they want. The steps are as follows:
+
+1.  The user specifies how many LP receipt tokens they would like to withdraw from.
+2.  OHM and counter-asset are withdrawn from the liquidity pool in exchange for the LP receipt tokens.
+3.  All the OHM received by the vault is burned.
+4.  All the received counter-asset and reward tokens are sent back to the user.
+
+## Benefits by Stakeholder
+
+### For the Protocol:
+- Facilitating the provision of liquidity for OHM pairs through a more streamlined process.
+- Increasing the liquidity of OHM pairs by incentivizing users to provide liquidity using selected, high-quality counter assets.
+- Dampening OHM volatility against these high-quality counter-pair assets, helping position OHM as a base asset with great liquidity routing.
+
+### For Partners:
+- A more efficient liquidity mining vehicle.
+- Olympus does not take a portion of the rewards provided by the partner protocol, meaning partners get 2x TVL for their rewards compared to traditional liquidity mining systems.
+- Since OHM is a volatile asset with decent price stability, by pairing with it partners can:
+    - Reduce the Impermanent Loss (IL) that Liquidity Providers (LPs) would accure when providing liquidity agains stables.
+    - Minimize the price supression that arbitrageurs can produce when their pools are paired with ETH.
+
+
+### For Users:
+- If the ratio of assets in the liquidity pool changes significantly over time, users may experience impermanent losses, which occur when the value of the assets they receive upon withdrawal is lower than the value of the assets they originally deposited.
+- To encourage users to provide liquidity and reduce the risks associated with liquidity provision, Olympus is giving up all of its rewards and allocating them to users instead.
+
+## Risks by Stakeholder
+
+### For the Protocol:
+-   The OHM is exposed to the counter asset's volatility.
+-   Newly minted supply could lead to OHM depreciation if the market is inefficient.
+
+### For Partners:
+-   The asset is exposed to OHM's volatility.
+-   If Olympus appreciates a lot in price, their asset could be impacted due to arbitrageurs. However, this risk is minimized by the fact that RBS (Reference Based Stabilization) is actively dampening OHM volatility versus DAI.
+
+### For Users:
+-   The typical liquidity provider opportunity costs apply in this mechanism too.
+-   If the composition of the pool changes over time, users may incur some Impermanent Loss (IL).
+-   Olympus gives up its rewards in favor of users to incentivize deposits and minimize the risks of liquidity provisioning.
+
+[yella's infographic on IL risk]
```

Comment:
just making sure this is TODO

Reply:
> just making sure this is TODO

Yes sir, this is the TODO list:
- infographics by Yella
- add real github links, which won't be public until we publish them on `olympus-v3`
- update the names according to the final project name (Liquidity AMO? Liquidity Engine?)
- anything which reviewers may spot